server/src/main/java/org/opensearch/rest/action/document/RestDeleteAction.java

```diff
@@ -50,17 +49,13 @@
 import static org.opensearch.rest.RestRequest.Method.DELETE;
 
 public class RestDeleteAction extends BaseRestHandler {
-    private static final DeprecationLogger deprecationLogger = DeprecationLogger.getLogger(RestDeleteAction.class);
-    public static final String TYPES_DEPRECATION_MESSAGE = "[types removal] Specifying types in "
-        + "document index requests is deprecated, use the /{index}/_doc/{id} endpoint instead.";
 
     @Override
     public List<Route> routes() {
         return unmodifiableList(
             asList(
                 new Route(DELETE, "/{index}/_doc/{id}"),
-                // Deprecated typed endpoint.
-                new Route(DELETE, "/{index}/{type}/{id}")
+                new Route(DELETE, "/{index}//{id}")
```

Comment:
Oops, I think  we need only one slash, right? `//` -> `/`

Reply:
Fixed.